deduplicator/src/lib/dedupe.rs

```diff
@@ -16,13 +16,47 @@ use crate::utils::{field_compare, opt_field_compare, postal_repr};
 const GEOHASH_PRECISION: u32 = 5;
 
 lazy_static! {
```

Comment:
Not linked to this PR but you should use `once_cell`. For example: https://github.com/GuillaumeGomez/libpostal-rs/blob/master/src/core.rs#L14-L15

Reply:
This should be a short change, I think I should do this in a new pull request to avoid to much digression here.

Reply:
Absolutely. That's why I added "not linked to this PR". ;)